.jenkins/pytorch/test.sh

```diff
@@ -51,6 +51,12 @@ else
   export PYTORCH_TEST_SKIP_NOARCH=1
 fi
 
+if [[ -n "$IN_PULL_REQUEST" ]]; then
+  export TEST_SKIP_CUDA_MEM_LEAK_CHECK=1
+else
+  export TEST_SKIP_CUDA_MEM_LEAK_CHECK=0
```

Comment:
```suggestion
  export PYTORCH_TEST_SKIP_CUDA_MEM_LEAK_CHECK=1
else
  export PYTORCH_TEST_SKIP_CUDA_MEM_LEAK_CHECK=0
```
Shouldn't these be `PYTORCH_TEST_SKIP_CUDA_MEM_LEAK_CHECK`?